pkg/expr/threshold.go

```diff
@@ -19,23 +19,29 @@ type ThresholdCommand struct {
 }
 
 const (
-	ThresholdIsAbove        = "gt"
-	ThresholdIsBelow        = "lt"
-	ThresholdIsWithinRange  = "within_range"
-	ThresholdIsOutsideRange = "outside_range"
+	ThresholdIsAbove                = "gt"
+	ThresholdIsBelow                = "lt"
+	ThresholdIsWithinRange          = "within_range"
```

Comment:
This is a small nit but could we have it so `ThresholdIsWithinRange` and `ThresholdIsOutsideRange` are below `ThresholdIsLe` as then the range consts will be together?